### ISSUE TITLE
Where did GetCallingTokenInfoAsync(), RenewTokenAsync() and RevokeTokenAsync() go to?

### ISSUE THREAD

**Issue by pvandervelde** [rajanadar/VaultSharp#77]
When the `VaultClient` was refactored these methods were moved or removed. How do we now get the calling token information, renew a token and revoke a token?

**Comment by pvandervelde** [comment]
@rajanadar Any suggestions as to where to look for these. We're wanting / needing to update to the latest version but without knowing how to update our code to point to these methods or their equivalents we can't renew / revoke tokens which makes having tokens pointless

**Comment by b1ff** [comment]
Hi @pvandervelde,
I'm not a maintainer, but I will try to help.
Currently after initial authentication every provider will emit token.
That token will be written to `ReturnedAuthInfo` field and also Token auth method will be set to Auth property.
Using Token you can run all of those operations. Next method are available:
```
var client = new VaultClient(...);
client.V1.Auth.Token.RenewSelfAsync(); // renew-self operation in vault
client.V1.Auth.Token.LookupSelfAsync();
client.V1.Auth.Token.RevokeSelfAsync();
```
The code above should work for each type of authentication, except `CustomAuthInfo`. For `CustomAuthInfo` fix is added in #81 
Hopefully it will help you.

**Comment by pvandervelde** [comment]
@b1ff Thanks for that! I will update my code and test that.
